extensions/amp-google-read-aloud-player/0.1/amp-google-read-aloud-player.js

```diff
@@ -89,17 +92,22 @@ export class AmpGoogleReadAloudPlayer extends AMP.BaseElement {
     this.play(false /** unusedIsAutoplay */);
   }
 
+  /** @return {string} */
+  getIframeBaseSrc_() {
+    const baseSrc = this.element.getAttribute(BASE_SRC_PARAM_NAME);
+    return baseSrc ?? DEFAULT_IFRAME_BASE_URL;
+  }
+
   /** @return {string} */
   getIframeSrc_() {
-    const src = addParamsToUrl(
-      IFRAME_BASE_URL,
-      getDataParamsFromAttributes(
-        this.element,
-        /* opt_computeParamNameFunc = */ undefined,
-        '^(.+)'
-      )
+    const dataParams = getDataParamsFromAttributes(
```

Comment:
nit: Just for diff's sake I would revert this method fully.

Reply:
Done.